gui/mozregui/pref_editor.py

```diff
@@ -30,13 +30,13 @@ def data(self, index, role=Qt.DisplayRole):
             if index.column() == 0:
                 return name
             else:
-                return repr(value)
+                return QString(value).__str__()
```

Comment:
Well, I would still like to see the quotes, so repr(value) is fine for me. What do you think of removing the first 'u' (if there is one) in the result of repr(value) ?